packages/stryker-typescript/test/unit/TypescriptConfigEditorSpec.ts

```diff
@@ -1,6 +1,6 @@
 import * as fs from 'fs';
 import * as path from 'path';
-import * as log4js from 'log4js';
+import * as log4js from 'stryker-api/logging';
```

Comment:
we call it log4js here and import the entire module

Reply:
Refactored the name, but the entire module is needed in order to mock away the `getLogger` method.